### PR TITLE
chore: 🤖 Bump binary version and update changelog

### PR DESCRIPTION
**Pull request by calcaide**
Bump binary version and update changelog.

### CODE REVIEW

File: ui/desktop/CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
 
-## Next [CLI 0.7.x] ()
+## v1.4.2 [CLI 0.7.6] (2022.03.16)
 
 ### New
 
 - Support dynamic host catalogs. ([PR](https://github.com/hashicorp/boundary-ui/pull/981))
+- Support SSH type targets. ([PR](https://github.com/hashicorp/boundary-ui/pull/1025))
```
Comment (cameronperera) [inline ui/desktop/CHANGELOG.md:10]: Is "Support SSH type targets" misleading here? Currently, we only display that the type is either TCP or SSH. No CRUD operations have been added yet.

Comment (calcaide) [reply ui/desktop/CHANGELOG.md:10]: The desktop client has no CRUD operations, just displays information.

Comment (cameronperera) [reply ui/desktop/CHANGELOG.md:10]: Oh right. Thanks for clarifying that. I was thinking big picture, not just the desktop client.

**Review by gsusmi — APPROVED** [review]
LGTM. Ty!

**Review by gsusmi — CHANGES_REQUESTED** [review]
Not to not include un-released features in changelog.

File: ui/desktop/CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (calcaide) [reply ui/desktop/CHANGELOG.md]: Deleted

**Review by cameronperera — APPROVED** [review]

**Pull request merged**